config/jobs/kubernetes-sigs/cluster-api/cluster-api-ci.yaml

```diff
@@ -47,6 +47,11 @@ periodics:
   interval: 1h
```

Comment:
Not directly related to this PR, but I'd really like to s/ci-/periodic-/ in the names of the periodic jobs.

Reply:
might as well throw it in with this PR